src/PKSim.Presentation/Services/EntityTask.cs

```diff
@@ -34,7 +37,7 @@ public ICommand Rename(IEntity elementToRename)
       private ICommand rename(IEntity elementToRename, bool isStructuralChange)
       {
          var dto = _renameObjectDTOFactory.CreateFor(elementToRename);
-         var newName = NewNameFor(elementToRename, dto.UsedNames, dto.ContainerType);
+         var newName = NewNameFor(elementToRename, dto.UsedNames, TypeFor(elementToRename));
```

Comment:
Just using the wrong tupe after refactoring. ContainerType is always project for bb